File: JS PS/q21.js
// 21. Write a program to convert a given string to uppercase.
// Example: Input: "hello", Output: "HELLO".

function capital(str) {
    console.log(str.toUpperCase());
}

capital('hello')


function capitalize(str) {
    let string = '';
    for (let i = 0; i < str.length; i++) {
        let charCode = str.charCodeAt(i);
        if (charCode >= 97 && charCode <= 122) {
            string += String.fromCharCode(charCode - 32);
        } else {
            string += str[i];
        }
    }
    console.log(string);
}

capitalize('hello');

File: Clang PS/Q21.c
#include <stdio.h>

int main() {
char str[]= "hello";

for (int i=0;i<str[i];i++){
    if(str[i] >= 'a' && str[i]<= 'z'){
        str[i]-=32;
    }
}
printf("%s\n",str);

    return 0;
}